examples/multi-agent-debate/config.py

```diff
@@ -0,0 +1,65 @@
+from rich.prompt import Prompt
+import langroid.language_models as lm
+import langroid.utils.configuration
+
+
+def get_global_settings(debug=False, nocache=True):
+    """
+    Returns global settings for Langroid.
+
+    Args:
+        debug (bool): Enable or disable debug mode.
+        nocache (bool): Enable or disable caching.
+
+    Returns:
+        Settings: Langroid configuration settings.
+    """
+    return langroid.utils.configuration.Settings(
+        debug=debug,
+        cache=not nocache,
+    )
+
+
+def get_base_llm_config(streamer=None):
+    """
+    Prompts the user to select a base LLM configuration.
+
+    Args:
+        streamer (Callable): Function to handle streaming tokens.
+
+    Returns:
+        OpenAIGPTConfig: Base configuration for the selected LLM.
+    """
+    chat_model_option = Prompt.ask(
+        "Which OpenAI Model do you want to use? Select an option:\n"
+        "1: GPT4o\n"
+        "2: GPT4\n"
+        "3: Mistral: mistral:7b-instruct-v0.2-q8_0a\n"
+        "Enter 1, 2, or 3:",
+        choices=["1", "2", "3"],
+        default="1"
+    )
+
+    model_map = {
+        "1": "gpt-4o",
```

Comment:
use `lm.OpenAIChatModel.GPT4o` etc here, so you get the right model names

Reply:
> use the actual openai mdl names here , e.g. `gpt-4`, `gpt-4o`, then directly use it to set `chat_model`, no need for `getattr`

do you recommend `lm.OpenAIChatModel.GPT4o` be used for `chat_model_option` instead of `model_map` or somewhere else?

Reply:
I just meant in the `model_map` -- "gpt-4o" is not an actual model name in langroid -- we use "gpt-4o-2024-11-20" to make sure it points to a fixed model. But if you use `lm.OpenAIChatModel.GPT4o` you can be sure it is a valid name